### ISSUE TITLE
Kill_chain_phase label "Weaponization" not working properly

### ISSUE THREAD

**Issue by dbuchberger** [splunk/security_content#960]
Maybe I am missing something but whenever I set the kill_chain_label tag to Weaponization in one of my detections, it's not displayed in the overview dashboard. No matter what - it will always display "0".

**Comment by dbuchberger** [comment]
okay, found the issue - the label needs to be named "Weaponize" so that it's mapped with the Weaponization kill chain phase.
btw is it intended that the story only shows the last CIS Control of the related detections? so e.g. when the detection has CIS 10, CIS 11 and CIS 12 the story will only have the CIS 12 displayed in its overview.

**Comment by patel-bhavin** [comment]
@dbuchberger : the tag to be used is "kill_chain_phases"
![image](https://user-images.githubusercontent.com/7771446/100805067-df7bf380-33e2-11eb-87ba-79f3917e44e6.png)

and tagging "kill_chain_phases" as  "Weaponization" should work just fine

**Comment by patel-bhavin** [comment]
Can you also perhaps share a screenshot of a story in ESCU where only the last CIS is displayed?  Thank you.

**Comment by dbuchberger** [comment]
@patel-bhavin are you sure? I tried the tag "Weaponization" but in the ESCU summary the Weaponization Kill chain phase did always show 0, when I changed the tag to "Weaponize" instead, it worked just fine. I also installed the app form the splunk app store and the "Weaponization" in the overview also displayed 0.

regarding the CIS issue, there is already a pull request from a colleague of mine and me - https://github.com/splunk/security-content/pull/962
The issue is that from the first search all the related cis controls are overwritten and only the last one will be taken, if there are any additional searches linked to the analytic story the cis controls of those searches will be added just fine.
